samples/csharp_dotnetcore/53.teams-messaging-extensions-action-preview/Bots/TeamsMessagingExtensionsActionPreviewBot.cs

```diff
@@ -143,17 +143,32 @@ protected override async Task<MessagingExtensionActionResponse> OnTeamsMessaging
             var activityPreview = action.BotActivityPreview[0];
             var attachmentContent = activityPreview.Attachments[0].Content;
             var previewedCard = JsonConvert.DeserializeObject<AdaptiveCard>(attachmentContent.ToString(), new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore });
-
+            previewedCard.Version = "1.0";
             var exampleData = AdaptiveCardHelper.CreateExampleData(previewedCard);
 
             // This is a send so we are done and we will create the adaptive card editor.
             var adaptiveCard = AdaptiveCardHelper.CreateAdaptiveCard(exampleData);
 
             var message = MessageFactory.Attachment(new Attachment { ContentType = AdaptiveCard.ContentType, Content = adaptiveCard });
 
+            //User Attribution for Bot messages
```

Comment:
Please change to "//Optional: Use if you want the Teams client to mark your message with the name of the user using the messaging extension"

Reply:
@clearab , I've made the optional use for User attribution message, Let me know if anything else needs to be added.